### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by dink** [AliasIO/Shot.io#1]
I was wondering how to make this run.

I followed all the instructions but I still can't make this app run -- Surely I'm missing something.

When fired up from the public directory on `http://127.0.0.1:3000` I'm getting the following:

http://127.0.0.1:3000/installinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstallinstall

What else is is supposed to be in the `config/main.php` file besides 'siteName'? 

Thank you.

**Comment by AliasIO** [comment]
Not sure what's happening there, perhaps the port number is throwing it off. Can you try running it on port 80?

**Comment by dink** [comment]
Ok, the port isn't apparently the problem. Running it on any port now and I'm greeted with this:

```
Exception: 9e8bd

The issue has been logged. Please contact the website administrator.
```

I'll try to run it on a virtual linux box later on to see if windows is crapping on me.

**Comment by AliasIO** [comment]
Find the actual error in `log/exceptions.log`. If it's not there, make the `log` folder writeable.

**Comment by dink** [comment]
Ok, here's a typical line from my log file:

`13ba0 Sat, 13 Jun 2015 14:39:06 +0000 SQLSTATE[HY000]: General error: 1 no such table: albums in D:\projects\shotio\vendor\Shot\Controllers\Index.php on line 35`

Any ideas?

**Comment by AliasIO** [comment]
Yeah the install script isn't running. You need to rewrite all URLs to `index.php?q=<query>` in your virtual host configuration. Are you using Apache or Nginx?

For now, you can also try going to `index.php?q=install` directly.

**Comment by dink** [comment]
I was just using the build-in php web server, but normally I use nginx and would appreciate the rewrite rules if you have them. 

Ok, now it says the installation is complete after attaching `index.php?q=install`, but the index page remains same as before with the same log error message.

**Comment by AliasIO** [comment]
Does `db/db.sdb` exist? You may need to make the `db` folder writable. You can also try to populate the database on the CLI using `db/schema.sql`.

I haven't touched this project in a while, you can tell it's not quite finished.
